Query.js

```diff
@@ -30,7 +30,8 @@ var FirestoreQuery_ = function (from, callback) {
     '<': 'LESS_THAN',
     '<=': 'LESS_THAN_OR_EQUAL',
     '>': 'GREATER_THAN',
-    '>=': 'GREATER_THAN_OR_EQUAL'
+    '>=': 'GREATER_THAN_OR_EQUAL',
+    'array-contains': 'ARRAY_CONTAINS'
```

Comment:
Instead of using `array-contains`, can we keep it short and simple with `contains`? I thought about using `in`, but when trying to read the code, having parameters `(field, 'in' value)` doesn't seem to make as much sense as `(field, 'contains', value)`.

Reply:
I have renamed the operator.
Please keep in mind to add documentation about this so users know that to use the ARRAY_CONTAINS they must use contains.